cmd/dockerd/docker.go

```diff
@@ -21,14 +20,14 @@ type daemonOptions struct {
 	version      bool
 	configFile   string
 	daemonConfig *config.Config
-	common       *cliflags.CommonOptions
+	common       *daemonFlags
```

Comment:
I guess this `common` field could also use a rename to something else, depending on what name we give the new struct